### DISCUSSION TITLE
part-ii/2014-103/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#208] [Question comments]
Part II 2014 Number Theory: Paper 3, Section II, F 

https://questions.tripos.org/part-ii/2014-103/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

This is almost entirely bookwork. The only part that is not bookwork is the last part.

The answer to the last part is "no". A counterexample is that $8$ is primitive root $\mod 3$ (since $8 \equiv 2 \pmod{3}$), but $8$ is not a primitive root $\mod 9$, since $8^2 \equiv 64 \equiv 1 \pmod{9}$.

</details>
